jsk_apc2016_common/launch/object_segmentation_3d.launch

```diff
@@ -4,16 +4,22 @@
   <arg name="INPUT_IMAGE" />
```

Comment:
Oops, maybe you need to stop changing current api of this launch file, because b4 students also use this file.
You should use `<arg default=xxx>`.